gatsby-config.js

```diff
@@ -183,6 +183,23 @@ module.exports = {
         trackingId: 'UA-162021632-1',
       },
     },
+    {
+      resolve: 'gatsby-plugin-google-tagmanager',
+      options: {
+        id: 'GTM-MPPD486',
+
+        // Include GTM in development.
+        // Defaults to false meaning GTM will only be loaded in production.
+        includeInDevelopment: true,
```

Comment:
@JessicaNovo I know you used this for testing locally. Should we remove this now that we're deploying, or is this still needed for GTM to work?